frontend/src/global.js

```diff
@@ -0,0 +1,12 @@
+const devBackend = "http://localhost:8000/api";
+const prodBackend = "https://localhost/api";
+
+const getAPIUrl = async () => {
+  if (window.mode === "dev") {
+    return devBackend;
+  } else {
+    return prodBackend;
+  }
+};
+
+export const BACKEND = await getAPIUrl();
```

Comment:
```suggestion
const getAPIUrl = () => {
  if (window.mode === "dev") {
    return devBackend;
  } else {
    return prodBackend;
  }
};

export const BACKEND = getAPIUrl();
```
이렇게 async없어도 될 것 같습니다!